atd-etl/afd_ems_import/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04
```

Comment:
this is another one of those cases where it would be nice to start from a python-based image, but i understand that might be a non-trivial change to make at this stage and i know we need to ship 🚢 🚢 🚢 🚢

Reply:
I think it can go either way on this one. If the python image is based on debian, then I'm 100% on board because it'd get you the fixed version of python, like we've talked about in the past and I'd be able to reuse the code below that installs the SSH daemon. 

Quick look at docker hub, and this is 100% the case: [3.9.17-bullseye](https://hub.docker.com/layers/library/python/3.9.17-bullseye/images/sha256-9c50b04e7ebc4c5a1afbf73786863d9e317414d5f2850d37cbb78fdc9742239f?context=explore). ✅

I'll work on porting this over this afternoon to a python image.

Reply:
This worked out super well - thanks for this awesome suggestion. I'm adopting this method going forward. If there's a hint of python on the image's task, start with a python image. You can still get 100% of debian based OS stuff right in there.

Reply:
niiice! thanks for getting this change in. really good stuff!